src/components/SelectDropdown/index.js

```diff
@@ -8,6 +8,12 @@ import Link from '../Link';
 
 class SelectDropdown extends React.Component {
   static propTypes = {
+    /**
+     * Button text object with label and current value
```

Comment:
I think you mean "content" value but maybe not? 
```suggestion
     * Button text object with label and content value
```